plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaTableHandle.java

```diff
@@ -54,6 +54,7 @@
     private final Optional<String> messageDataSchemaLocation;
     private final Optional<String> keySubject;
     private final Optional<String> messageSubject;
+    private final Optional<List<KafkaColumnHandle>> keyColumns;
```

Comment:
Previously this was a string which encoded <column name>:<column type> and was decoded multiple times which is unnecessary.

Reply:
Changed this to `keyColumn` as the usage is to have a message subject stored in schema registry but a raw key, like a long or varchar id. The only way to specify the type (that I am aware of) is to encode it in the table name:
ex. 
```
SELECT col FROM kafka_table&key-column=my_id:varchar
```
Also added this to the exception message if it's incorrectly specified.